### ISSUE TITLE
Reduce mention of EPUB in the principles document

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/publ-a11y#32]
The principles document should be independent of metadata formats and publication formats. Following this principle we have removed mention of EPUB and EPUB accessibility conformance hyperlink in definitions, but the introduction still has mention of EPUB. For example:

"Metadata found either inside an EPUB or in its corresponding ONIX file may have important accessibility information that will help end users find and determine if this EPUB can meet their specific accessibility needs."

it can be modified to:
Metadata found either inside a digital publication or in its corresponding ONIX file may have important accessibility information that will help end users find and determine if this publication can meet their specific accessibility needs.

**Comment by madeleinerothberg** [comment]
Should we also remove "ONIX" in preparation for eventually including other non-embedded metadata formats? Is there a better industry-specific term for metadata held in a separate file?

**Comment by gregoriopellegrino** [comment]
What about "supply chain metadata"? But it doesn't fit MARC...

**Comment by avneeshsingh** [comment]
Or simply state external file

Metadata found either inside a digital publication or in its corresponding external file may have important accessibility information that will help end users find and determine if this publication can meet their specific accessibility needs.

**Comment by mattgarrish** [comment]
We've typically use "metadata record" or "external record" for this in the core specs. The metadata may not always be in a "file" when transmitted outside an epub.
